app/src/main/java/ch/epfl/sweng/erpa/services/GameService.java

```diff
@@ -12,16 +12,12 @@
 import lombok.NoArgsConstructor;
 import lombok.NonNull;
 
-public interface GameService {
-    String GAME_UUID_PREFIX = "game|";
-    String PROP_INTENT_GAMEUUID = "game uuid";
-
-    Optional<Game> getGame(String gid);
-
-    Set<Game> getAll();
+public interface GameService extends DataService<Game> {
+    String PROP_INTENT_GAME_UUID = "game|";
```

Comment:
`PROP_INTENT_GAME_UUID` It's supposed to be the “key” in the intent to search for the GameUuid. `game|`=`game|xxx-xxxx...` seems like a very odd encoding, can you leave the `GAME_UUID_PREFIX` and use a different value for `PROP_INTENT_GAME_UUID`? (Pref no spaces)